cmd/admin-replicate-update.go

```diff
@@ -48,7 +48,10 @@ var adminReplicateUpdateFlags = []cli.Flag{
 		Usage:  "enable synchronous replication for this target, valid values are ['enable', 'disable'].",
 		Value:  "disable",
 		Hidden: true, // deprecated Jul 2023
-
+	},
+	cli.StringFlag{
+		Name:  "bucket-bandwidth",
+		Usage: "Set default bandwidth limit for bucket in bits per second (K,B,G,T for metric and Ki,Bi,Gi,Ti for IEC units)",
```

Comment:
Please add help entries for these flags like `bucket-bandwidth`, `mode`, `sync` etc